App.js

```diff
@@ -1,26 +1,25 @@
 import React from 'react'
-import styled from 'styled-components/native'
+import { NavigationContainer } from '@react-navigation/native';
+import { createStackNavigator } from '@react-navigation/stack';
 
-const Container = styled.View`
-  flex: 1;
-  background-color: papayawhip;
-  justify-content: center;
-  align-items: center;
-`
+import { LatestNews } from './components/LatestNews'
+import { Home } from './components/Home'
 
-const Title = styled.Text`
-  font-size: 24px;
-  color: palevioletred;
-`
+const Stack = createStackNavigator();
 
 const App = () => {
   return (
-    <Container>
-      <Title>This is your cool app!</Title>
-      <Title>Go to App.js and start coding</Title>
-      <Title>💅💅💅</Title>
-    </Container>
+    <>
```

Comment:
No need to use React Fragment here since you return only one element (`<NavigationContainer />`)